sysdig/resource_sysdig_monitor_alert_v2_common.go

```diff
@@ -180,36 +211,50 @@ func buildAlertV2CommonStruct(d *schema.ResourceData) (alert *monitor.AlertV2Com
 
 		for _, channel := range attr.(*schema.Set).List() {
 			channelMap := channel.(map[string]interface{})
+			channelID := channelMap["id"].(int)
+			nc, err := client.GetNotificationChannelById(ctx, channelID)
```

Comment:
I meant the `buildAlertV2*` functions to be mappers between the `ResourceData` and the `monitor.AlertV2*` structs. Can the validation logic be moved later on in the `sysdigMonitorClient`?

Reply:
ok, i moved the logic on the client